server/client/ruby/cpc

```diff
@@ -26,8 +26,9 @@ class CandlepinCLI < Candlepin
 
   private
 
-  def self.callable_name?(method_name)
-    !method_name.end_with?('=')
+  def self.callable_name?(method)
+    method_name = method.to_s
+    return !['?', '='].include?(method_name[-1])
```

Comment:
You should probably remove the "?".  Methods ending in a question mark are callable.  They just, by convention, return booleans.  You could probably just get away with

```
!method_name.to_s.end_with?('=')
```